stellarphot/settings/models.py

```diff
@@ -0,0 +1,48 @@
+# Objects that contains the user settings for the program.
+
+from enum import Enum
+
+from astropy import units as un
+
+from pydantic import BaseModel, Field, conint, root_validator
+
+from .autowidgets import CustomBoundedIntTex
+
+__all__ = ['ApertureSettings']
+
+class ApertureUnit(Enum):
+    PIXEL = un.pix
+    ARCSEC = un.arcsec
+
+
+class ApertureSettings(BaseModel):
+    """
+    Settings for aperture photometry.
+
+    Parameters
+    ----------
+
+    radius : int
+        Radius of the aperture in pixels.
+
+    inner_annulus : int
+        Inner radius of the annulus in pixels.
+
+    outer_annulus : int
+        Outer radius of the annulus in pixels.
+    """
+    radius : conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
+    gap : conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
+    annulus_width : conint(ge=1) = Field(autoui=CustomBoundedIntTex, default=1)
```

Comment:
So these have to be integers constrained to be greater or equal to 1?  Also, shouldn't the default radius be like 5, same for gap and annulus?

Reply:
To repeat my comment from the photometry.py, should we include a sample declaration of the ApertureSettings object in the documentation?  Not sure how familiar people are with data classes let alone pydantic data classes.

Reply:
Done in the latest revision (also helped me catch an error in the documentation, which still said to use inner/outer annulus instead of gap and width)